services/madoc-ts/src/frontend/site/features/CanvasViewerGrid.tsx

```diff
@@ -39,15 +39,21 @@ export const CanvasViewerControls = styled(ButtonRow)`
   top: 0;
   right: 1em;
   z-index: 20;
+  display: flex;
+  flex-direction: column;
 `;
 
 export const CanvasViewerButton = styled(Button)`
-  border-radius: 0;
+  border-radius: 3px;
   padding: 0.8em;
   font-size: 1em;
   border: none;
-  background: #fff;
+  background: rgba(255, 255, 255, 0.5);
+  color: #002d4b;
   display: flex;
+  margin-left: 0;
+  margin-bottom: 0.5em;
```

Comment:
Does this change the existing styling?

Reply:
Yeah I just thought it looked better stacked and not white but I can change it back!

<img width="84" alt="Screenshot 2022-11-08 at 16 35 04" src="https://user-images.githubusercontent.com/52662841/200623487-6ded0856-12b2-4ecc-9331-cb301514a268.png">
<img width="105" alt="Screenshot 2022-11-08 at 16 34 55" src="https://user-images.githubusercontent.com/52662841/200623497-a23e3e44-c6ff-4040-a077-3f7c6c7f3930.png">

Reply:
Yeah I think it looks good, definitely worth having as an option. One small issue might be the order. Usually zoom in will be above zoom out, might be a flex property we can use to flip the order.

Reply:
Yeah, good point. I'm sure theres a flex order property